commit d221d41575392435225186ef6c3f93041dcd293a
Author: Alexandre Frade <admfrade@gmail.com>
Date:   Fri May 10 14:32:50 2019 -0300

    mm: set 2 megabytes for address_space-level file read-ahead pages size
    
    Signed-off-by: Alexandre Frade <admfrade@gmail.com>

diff --git a/include/linux/mm.h b/include/linux/mm.h
index cfaa8feecfe8..452eb0ae12d2 100644
--- a/include/linux/mm.h
+++ b/include/linux/mm.h
@@ -2406,7 +2406,7 @@ int __must_check write_one_page(struct page *page);
 void task_dirty_inc(struct task_struct *tsk);
 
 /* readahead.c */
-#define VM_READAHEAD_PAGES	(SZ_128K / PAGE_SIZE)
+#define VM_READAHEAD_PAGES	(SZ_2M / PAGE_SIZE)
 
 int force_page_cache_readahead(struct address_space *mapping, struct file *filp,
 			pgoff_t offset, unsigned long nr_to_read);
